### DISCUSSION TITLE
Documentation of Python and grpc API not in line with git

### DISCUSSION THREAD

**Discussion by awolff64** [coreemu/core#758] [General]
The examples for the APIs in the documentation differ from the actual examples (at least on did not compile).
The link to the examples is no more valid.

Just a thought:
Remove the examples code from the documentation and point to the related example file? Could make maintenance easier.

Thanks for the great work.

**Comment by bharnden** [comment]
Can you specify exactly what is different? So it can be corrected.

I agree with you it is unideal, I am currently looking into a documentation revamp, that may help in this area.

**Reply by awolff64** [reply]
The links to the example files for python API and grpc API are no more valid.
I cut and pasted one example from the WEB page and got errors when I tried to run it. 
I then found the example files under package and it did the job.
Do not remember which example it was.
The examples under package all worked fine.

Hope this helps, regards
